### PR TITLE
Add project description to README

### PR DESCRIPTION
**Pull request by Solis13**
I have created a README.md for the project. Please review it and let me know if any additional information should be added or if anything should be removed. Thank you! @littlestripes @MAvecillas23

### CODE REVIEW

**Review by littlestripes — COMMENTED** [review]
What you've written looks good. One thing I would recommend adding would be more detailed instructions for running the app. Something like:

```md
Dependencies: Python >=3.12

1. Run `python -m venv venv && source venv/bin/activate && pip install -r requirements.txt`.
2. Set the `[WHATEVER_THE_API_KEY_ENVIRONMENT_VARIABLE_NAME_WILL_BE]` environment variable to the National Flood Data API key.
3. Run `python app.py` and access `http://localhost:5000`.
```

We can add separate instructions for Windows, too. They'll be pretty similar, I just don't remember them off the top of my head.

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+pollution for that area.
+
+Features
+
+    1.Earthquake Data
+
+    -Description: Retrieves earthquake information within a 15 mile radius of the specified location.
+    
+    -Data Returned: 
+        -Date of each earthquake
```
Comment (MAvecillas23) [inline Readme.md:13]: Add location of the earthquake within the cities radius

Comment (Solis13) [reply Readme.md:13]: I've just updated it. Let me know if it looks good

File: Readme.md [hunk 2]
```diff
@@ -46,4 +47,20 @@ Installation
...
+        pip install -r requirements.txt
+2. Set the export [API_KEY_ENVIRONMENT_VARIABLE_NAME]=your_api_key_here
+3. Run `python app.py` and access `http://localhost:5000`.
+
+Windows:
+1. Run python -m venv venv
+   venv\Scripts\activate
+   pip install -r requirements.txt
+
+2. set [API_KEY_ENVIRONMENT_VARIABLE_NAME]=your_api_key_here
```
Comment (MAvecillas23) [inline Readme.md:65]: One last thing, I'd specify you create two environment variables. One for flooding and another for air pollution

**Comment by MAvecillas23** [comment]
looks good

**Comment by claraj** [comment]
What are you using that requires Python 3.12, or is unsupported in earlier versions?

**Pull request merged**